config.lua.dist

```diff
@@ -161,6 +161,10 @@ removeOnDespawn = true
 walkToSpawnRadius = 15
 monsterOverspawn = false
 
+-- Bestiary
+bestiaryPartyKillSharing = true
+bestiaryPointsPerKill = 1
```

Comment:
I don't think we need these options. The code is easily editable if someone wants special behavior, and it just creates configuration creep - more configuration that almost no one will change, and if they need to change, they have to change some of the code anyway.